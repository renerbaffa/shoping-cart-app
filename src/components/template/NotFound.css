.NotFound-container {
  display: flex;
  justify-content: center;
  font-size: 26px;
  padding-top: 12px;
}