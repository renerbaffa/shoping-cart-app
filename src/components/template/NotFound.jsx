import React from 'react';

import './NotFound.css';

const NotFound = () => (
  <div className="NotFound-container">
    <b>Page not found</b>
  </div>
);

export default NotFound;
