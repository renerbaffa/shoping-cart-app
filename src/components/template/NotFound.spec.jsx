import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from './NotFound';

describe('<NotFound />', () => {
  it('should show spinner correctly', () => {
    const notFound = renderer.create(<NotFound />).toJSON();
    expect(notFound).toMatchSnapshot();
  });
})