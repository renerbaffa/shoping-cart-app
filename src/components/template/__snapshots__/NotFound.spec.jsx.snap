// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should show spinner correctly 1`] = `
<div
  className="NotFound-container"
>
  <b>
    Page not found
  </b>
</div>
`;
